amazon/ion/writer_text.py

```diff
@@ -81,13 +73,14 @@ def _serialize_scalar_from_string_representation_factory(type_name, types, str_f
     def serialize(ion_event):
         value = ion_event.value
         validate_scalar_value(value, types)
-        return six.b(str_func(value))
+        # TODO latin-1 is what six.b was doing. it does not seem right to me either
+        return str_func(value).encode("latin-1")
```

Comment:
I agree... what happens if we change to UTF-8?

Reply:
I will _try_ that. But I confess this comment is really a stand-in for my not fully understanding the relevance of the code we have for handling unicode if we're only supporting python3. All Python3 strings are already unicode, so I'm not sure what we need to worry about.

I _think_ the answer is that Ion supports some characters that are outside the range of valid UTF-8. I would really appreciate any context you had here.

Reply:
After analyzing the code this is _not_ used for text. It is used for numbers and timestamps. The nice part about `encode()` is that will error if the codepoints cannot be encoded in the encoding.

I'm going to set it to UTF-8 with a comment. Likewise for above. Thanks for checking me on this.

Reply:
Sorry I didn't see your reply before re-replying. Will follow up with you offline to clarify my understanding further.